src/middleware/cache/index.test.ts

```diff
@@ -31,11 +31,23 @@ describe('Cache Middleware', () => {
   })
 
   app.use('/wait2/*', cache({ cacheName: 'my-app-v1', wait: true, cacheControl: 'max-age=10' }))
-  app.use('/wait2/*', cache({ cacheName: 'my-app-v1', wait: true, cacheControl: 'max-age=10' }))
+  app.use('/wait2/*', cache({ cacheName: 'my-app-v1', wait: true, cacheControl: 'max-age=20' }))
```

Comment:
If `max-age=20` is `Max-age=20`, the test will fail. If it compares the names of the directives, I think we have to make all uppercase letters lowercase first.

If there is a cost to lowercase, you might prefer to lowercase everything first, even if only `directives`.